main.tf

```diff
@@ -261,9 +261,7 @@ resource "azurerm_kubernetes_cluster" "main" {
     }
   }
   dynamic "api_server_access_profile" {
-    for_each = var.api_server_authorized_ip_ranges != null || var.api_server_subnet_id != null ? [
-      "api_server_access_profile"
-    ] : []
+    for_each = var.api_server_authorized_ip_ranges != null || var.api_server_subnet_id != null ? ["api_server_access_profile"] : []
```

Comment:
Is this a change in formatting that was applied automatically by `terraform fmt` ?